runtime/queries/python/highlights.scm

```diff
@@ -81,22 +96,23 @@
 (identifier) @variable
 
 ; Literals
-
+(none) @constant.builtin
 [
-  (none)
   (true)
   (false)
-] @constant.builtin
+] @constant.builtin.boolean
 
 (integer) @constant.numeric.integer
 (float) @constant.numeric.float
 (comment) @comment
 (string) @string
 (escape_sequence) @constant.character.escape
 
+["," "." ":" ";" (ellipsis)] @punctuation.delimiter
 (interpolation
   "{" @punctuation.special
```

Comment:
This seems wrong to me? We have an embedded tag specifically to match interpolation, but you removed it here.